lxd/db/storage_pools.go

```diff
@@ -111,9 +113,17 @@ SELECT storage_volumes.name, storage_volumes.type, projects.name, storage_volume
 		// Handle instances.
 		if r.volType == StoragePoolVolumeTypeContainer || r.volType == StoragePoolVolumeTypeVM {
 			if r.projectName == "default" {
-				usedby = append(usedby, fmt.Sprintf("/1.0/instances/%s", r.volName))
+				if allNodes && r.nodeID != nil && nodesName[*r.nodeID] != "none" {
```

Comment:
This would've been good to use the `shared/api` URL builder package too.

Reply:
I've just put up a PR for this :+1: https://github.com/lxc/lxd/pull/9888

Reply:
Thanks!